### PR TITLE
lightning: improve unit test coverage of 'lightning' package

### PR DESCRIPTION
**Pull request by lonng**
Signed-off-by: Lonng <chris@lonng.org>

### What problem does this PR solve? 

Improve unit test coverage of 'lightning' package

### CODE REVIEW

**Comment by lonng** [comment]
@kennytm @july2993 PTAL

File: lightning/log/log_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+func (s *logSuite) TestConfigAdjust(c *C) {
+	cfg := &log.Config{}
+	cfg.Adjust()
+	c.Assert(cfg.Level, Equals, "info")
+
+	cfg.File = "."
+	err := log.InitLogger(cfg, "info")
+	log.L().Named("xx")
+	c.Assert(err, ErrorMatches, "can't use directory as log file name")
```
Comment (july2993) [inline lightning/log/log_test.go]: put it right after L37

Comment (lonng) [reply lightning/log/log_test.go]: addressed

**Pull request merged**